Plots/Contours/NCL_polar_1_lg.py

```diff
@@ -0,0 +1,115 @@
+
+"""
+NCL_polar_1_lg.py
+================
+
+This script illustrates the following concepts:
+    - Drawing black-and-white contours over a polar stereographic map
+    - Drawing the northern hemisphere of a polar stereographic map
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/polar_1_lg.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/polar_1_lg.png
+"""
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import matplotlib.path as mpath
+import geocat.datafiles as gdf
+import matplotlib.ticker as mticker
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc"))
+U = ds.U[1, :, :]
+
+###############################################################################
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
+wrap_U = gvutil.xr_add_cyclic_longitudes(U, "lon")
+
+###############################################################################
+#Plot:
+    
+# Generate figure (set its size (width, height) in inches)
+fig = plt.figure(figsize=(10, 10))
+
+# Generate axes, using Cartopy, drawing coastlines, and adding features
+fig = plt.figure(figsize=(10, 10))
+projection = ccrs.NorthPolarStereo()
+ax = plt.axes(projection=projection)
+ax.coastlines(linewidths=0.5)
```

Comment:
I'm not sure if this matters, but in the original NCL plot the land is not outlined.

Reply:
Good catch, it matters.

Reply:
You could turn coastlines off. I recommend doing this so they don't distract from the contour lines.

Reply:
Switching them off now. Do you mind also posting the code that fixes the latitude line drop out at 150E? I will fix both issues and then upload the new file

Reply:
I don't think I caught what that fix was. @erogluorhan Did you know how to fix the latitude line drop out at 150E?

Reply:
I think that both of latitudinal and longitudinal line drop out issues were resolved during the previously closed ancestor PR of this PR (PR #68)? So, the latest version of the plot looks like it is already resolved:

![image](https://user-images.githubusercontent.com/32553057/83081726-8fd8fb80-a03e-11ea-942a-426ff7407a6d.png)

If you are explicitly looking for the exact codes that were the solution of this issue, I can try to grab them from commits of PR #68. Please let me know.

Reply:
The Longitude labels on the left hand side of the graph seem to be out of position when compared to the right hand side. It's just a kerning issue, so we may not need to address it. 30˚ and 150˚ W seem to be the worst, while 90˚ W is fine.

Reply:
Thanks for bringing this up, @pilotchute! Indeed, we are manually adjusting those labels and their positions as a walk-around to the labeling issue with circularly-bounded `ccrs.NorthPolarStereo() `projections. Therefore, I believe that this issue of label positioning could be solved by playing with the following codes:

```
# Manipulate longitude labels (0, 30 E, 60 E, ..., 30 W, etc.)
ticks = np.arange(0, 210, 30)
etick = ['0'] + [r'%d$^\circ$E' %
                 tick for tick in ticks if (tick != 0) & (tick != 180)] + ['180']
wtick = [
    r'%d$^\circ$W' %
    tick for tick in ticks if (
        tick != 0) & (
            tick != 180)]
labels = etick + wtick
xticks = [-0.8, 28, 58, 89.1, 120, 151, 182.9, -36, -63, -89, -114, -140]
yticks = [-3] + [-2] + [-1] + [-1] * 2 + [-1] + [-3] + [-7] + [-7] * 3 + [-7]


for xtick, ytick, label in zip(xticks, yticks, labels):
    ax.text(xtick, ytick, label, transform=ccrs.Geodetic())
```

@michaelavs , does that make sense?

Reply:
@erogluorhan I will look into the position they should be in, I had tweaked it a bit previously, so I think it will be a fairly simple fix. Also, I was looking through the commits from the previous version of this PR and couldn't find where the code was updated, do you mind looking into where it may be? I can then add it for when I commit the longitude label changes as well

Reply:
Hi @michaelavs I walked through each of those commits in PR #70 and #68, but there is nothing related to this issue. Indeed, I now believe that there should not be any manipulation of our own to overcome such an issue because the code that manages those latitude and longitude lines is the following:

```
# Manipulate latitude and longitude gridline numbers and spacing
gl.ylocator = mticker.FixedLocator(np.arange(0, 90, 15))
gl.xlocator = mticker.FixedLocator(np.arange(-180, 180, 30))
```

So, it should all be managed by `matplotlib.ticker`.

I doubt here whether the very first image with those line drop out issue was taken by a screenshot application, and those "drop-outs" were only a resolution artifact. Can you confirm whether you have observed those artifacts on the originally generated plot?

Reply:
@erogluorhan I just tried to run the original code you had sent me when we first went over the lat/lon issue to see if I had made an edit that would have caused this issue, and the line drop out is occurring at 150E in the original as well. You're able to run the code and there isn't a line dropout correct? If so, there may just be something wrong on my side that isn't necessarily code related that is causing the output to drop the line at 150E

Reply:
@michaela Yes, I tried the original code I had sent you initially, and it generates the following plot without any lat/lon drop-out:

![image](https://user-images.githubusercontent.com/32553057/83206179-8c607580-a10d-11ea-8696-22a52350eee0.png)

Could you confirm that you are running `make html` command and checking the plot from where it generates (_build/html/gallery/Contours)?